aldryn_forms/templatetags/aldryn_forms_tags.py

```diff
@@ -7,6 +7,18 @@
 register = template.Library()
 
 
+@register.assignment_tag(takes_context=True)
+def render_notification_text(context, email_notification, email_type):
```

Comment:
If "email_notifications" is optional why new `render_notification_text` it's in the main `aldryn_forms` templatetags namespace?